src/proto/mod.rs

```diff
@@ -175,6 +211,18 @@ impl<S: Read + Write> Client<S> {
             hello.labels = self.opts.labels.clone();
         }
 
+        single::write_command_and_await_ok(&mut self.stream, &hello)
+    }
+
+    #[cfg(feature = "ent")]
+    fn init_tracker(&mut self) -> Result<(), Error> {
```

Comment:
Why have this as a separate function when it is (as far as I can tell?) exactly identical to `init`?

Reply:
There is a whole block in `init` dedicated to a non-producer client (to a consumer basically), where we populate postname, wid etc fields, which is not needed for the `tracker`, since a it's not a producer, neither a consumer.

Reply:
Ohh, I see. That's fine. I just figured it might be an opportunity to share some code, but not terribly important in this case.